modules/sql/shared/src/test/scala/SqlWorldSuite.scala

```diff
@@ -1559,6 +1559,9 @@ trait SqlWorldSuite extends CatsEffectSuite {
       assertIO(resTotal, Some(expected))
   }
 
+  // JS treats floats as doubles
+  def isJS = Option(System.getProperty("java.vm.name")).contains("Scala.js")
```

Comment:
This was highly annoying but does the trick ...

Reply:
Ouch, but :+1: